internal/pipe/git/git.go

```diff
@@ -123,6 +130,10 @@ func validate(ctx *context.Context) error {
 	return nil
 }
 
+func getBranch() (string, error) {
+	return git.Clean(git.Run("rev-parse", "--abbrev-ref", "HEAD", "--quiet"))
```

Comment:
I think there are no tests checking that this gets the correct output...

Reply:
This is the typical command I use to get the current branch. But you're right, I don't know how to mock a test for this. Some hints would be welcome. But I know it works. :smile: 

```
[git/goreleaser] $ git rev-parse --abbrev-ref HEAD --quiet
branch_template
```

Reply:
Yeah, it works, but as-is if we remove the method, or change it to something that doesn't work, no tests will break..

here's an example: https://github.com/goreleaser/goreleaser/blob/master/internal/pipe/git/git_test.go#L30-L42

Reply:
That's great, thanks.